importer/importer.go

```diff
@@ -124,13 +124,12 @@ func (im *Importer) createCurl(sql string) string {
 	return curl
 }
 
-func (im *Importer) createDB(conn driver.Conn, db string) error {
-	sql := "CREATE DATABASE IF NOT EXISTS " + db + ";"
-	err := conn.Exec(context.Background(), sql)
-	return err
+func (im *Importer) createDB(conn driver.Conn, cluster, db string) error {
+	sql := "CREATE DATABASE IF NOT EXISTS " + db + " ON CLUSTER " + cluster + ";"
```

Comment:
Let's do a check here for whether `cluster == ""`. If the cluster is blank then we should omit this.